streamlit.py

```
@@ -10,42 +10,139 @@
 from enum import StrEnum
 from functools import partial
 from pathlib import PosixPath
-from typing import cast
+from typing import cast, Any
+import json
```

Ruff (0.8.0)

`json` imported but unused

Remove unused import: `json`

(F401)